tests/integration/security/util/authn/authn_util.go

```diff
@@ -38,12 +47,11 @@ func (c *TestCase) CheckAuthn() error {
 			err = results.CheckOK()
 		}
 		if err != nil {
-			return getErrorMessage(c.Request, "authenticated (code 200)", err.Error())
+			return fmt.Errorf("%s: want 200 authenticated, got %s", c, err.Error())
```

Comment:
does the `want`  already part of c.String()?

Reply:
Yeah, removed the `want ...` from the output.